data-connection-jdbc/build.gradle

```diff
@@ -21,33 +21,18 @@ dependencies {
 
     testImplementation mnReactor.micronaut.reactor
 
-    testImplementation(mnTestResources.testcontainers.mysql)
```

Comment:
These were probably all copied from data-jdbc and this module uses only h2 and postgres so remove unused dependencies.